ros2component_fixtures/CMakeLists.txt

```diff
@@ -0,0 +1,60 @@
+cmake_minimum_required(VERSION 3.5)
+
+project(ros2component_fixtures)
+
+# Default to C++14
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
+
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic)
+endif()
+
+find_package(ament_cmake REQUIRED)
+find_package(rclcpp REQUIRED)
+find_package(rclcpp_components REQUIRED)
+find_package(std_msgs REQUIRED)
+
+include_directories(include)
+
+# create ament index resource which references the libraries in the binary dir
+set(node_plugins "")
+
+add_library(talker_component SHARED
+  src/talker_component.cpp)
+target_compile_definitions(talker_component
+  PRIVATE "ROS2COMPONENT_FIXTURES_BUILDING_DLL")
+ament_target_dependencies(talker_component
+  "rclcpp"
+  "rclcpp_components"
+  "std_msgs")
+rclcpp_components_register_nodes(talker_component "ros2component_fixtures::Talker")
+set(node_plugins "${node_plugins}ros2component_fixtures::Talker;$<TARGET_FILE:talker_component>\n")
+
+add_library(listener_component SHARED
+  src/listener_component.cpp)
```

Comment:
@BMarchi do we need a second component at all to test the CLI?

Reply:
I prefer to have a second one with a different name rather than some weird stuff that could happen using just one component.

Reply:
What weird stuff?

Reply:
I don't know, for some reason the cli could clone nodes or it's ignoring new ones, for me is better to be prepared for the worst and I don't see anything bad of having just one extra node.